crates/fluvio-controlplane-metadata/src/partition/status.rs

```diff
@@ -38,6 +38,7 @@ pub struct PartitionStatus {
     #[fluvio(min_version = 5)]
     pub size: i64,
     pub is_being_deleted: bool,
+    pub base_offset: i64,
```

Comment:
Same here:
 ```suggestion
    #[fluvio(min_version = 15)]
    pub base_offset: i64,
```

And bump version of `impl Request for ObjectApiListRequest`

Reply:
Done! Thanks for the help!